File: yh_dual/src/yh_dual_time.cpp
#include "ros/ros.h"
#include "yh_dual/YhDual.h"

void msgCallback(const yh_dual::YhDual::ConstPtr& msg)
{
    ROS_INFO("%d.%d", msg->stamp.sec, msg->stamp.nsec);
}

int main(int argc, char** argv)
{
    ros::init(argc, argv, "yh_dual_time");
    ros::NodeHandle nh;

    ros::Subscriber sub = nh.subscribe("yh_dual_topic", 100, msgCallback);

    ros::spin();

    return 0;
}

File: yh_dual/scripts/yh_dual_time.py
#!/usr/bin/python3
# -*- coding: utf-8 -*-
import rospy
from yh_dual.msg import YhDual

def msgCallback(msg):
    rospy.loginfo(f"{msg.stamp.secs}.{msg.stamp.nsecs}")

def yh_dual_time():
    rospy.init_node("yh_dual_time")
    rospy.Subscriber("yh_dual_topic", YhDual, msgCallback, queue_size=10)
    rospy.spin()

if __name__ == "__main__":
    yh_dual_time()